sylviawanjiku/StoreManager
# #161044085 Store attendants should be able to add products to specific categories.

#### What does this PR do?
Allows store attendants to add products to their categories
#### Description of Task to be completed?
The products page has a column for categories that can be added by the admin on adding the item in the inventory in case the owner failed to add the store attendant can add by selecting from the drop down
#### How should this be manually tested?
On products page if empty the user should click on category to navigate to the add category page where you can select the category as per the item
#### Any background context you want to provide?
The admin can add the category when adding the product to inventory or any user can add the category from the products page
#### What are the relevant pivotal tracker stories?
#161044085
https://www.pivotaltracker.com/story/show/161044085
#### Screenshots (if appropriate)
![screenshot_2018-10-12 store manager](https://user-images.githubusercontent.com/28457081/46845861-ef40b400-cde5-11e8-9c6a-43141b2746a4.png)
![screenshot_2018-10-12 store manager products 1](https://user-images.githubusercontent.com/28457081/46845819-c6b8ba00-cde5-11e8-910f-4843c2e70e47.png)
![screenshot_2018-10-12 store manager add cart](https://user-images.githubusercontent.com/28457081/46845828-cf10f500-cde5-11e8-9094-84b8c6fedb38.png)

#### Questions:
